org.adempiere.base/src/org/compiere/print/ServerReportCtl.java

```diff
@@ -168,6 +168,7 @@ public static boolean runJasperProcess(int Record_ID, ReportEngine re, boolean I
 		}
 		jasperProcessInfo.setRecord_ID ( Record_ID );
 		jasperProcessInfo.setTable_ID(printInfo.getAD_Table_ID());
+		jasperProcessInfo.setTransientObject(format);

```

Comment:
this is insufficient, check the references for ServerProcessCtl.process(ProcessInfo pi, Trx trx) in Eclipse (References > Workspace) and there are many other place calling this.

Reply:
you mean to move the setTransientObject part elsewhere, or setting the format in ServerProcessCtl.process should be move somewhere else?

Reply:
I means with the approach you are using here, you need to duplicate the setTransientObject call for all the code that's calling ServerProcessCtl.process(ProcessInfo pi, Trx trx).

Reply:
i see thanks, i've committed the fix with the other two small changes